doc/source/guide/spatial-partitioning.ipynb

```diff
@@ -0,0 +1,383 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "aca93ab3-f452-4725-b864-b41cd0327954",
+   "metadata": {},
+   "source": [
+    "# Spatial partitioning in Dask-GeoPandas\n",
+    "\n",
+    "Dask-GeoPandas has implemented `spatial_shuffle` to repartition `Dask.GeoDataFrames`.\n",
+    "For those who are not familiar with Dask, a Dask DataFrame is internally split into many partitions, where each partition is a Pandas DataFrame.\n",
+    "Partitions are split vertically along the index; \n",
+    "\n",
+    "<img src=\"https://docs.dask.org/en/latest/_images/dask-dataframe.svg\" alt=\"drawing\" width=\"300\"/>\n",
+    "\n",
+    "\n",
+    "Partitioning a Dask DataFrame avoids element-wise operations, where every possible candidate in a query is evaluated.\n",
+    "By evaluating less candidates, we reduce memory overhead and the time taken to execute a query.\n",
```

Comment:
What are you trying to say here? I am not sure it is correct.

Reply:
Removed - I think this was copied from previous work.